src/libsystemd/meson.build

```diff
@@ -158,6 +158,10 @@ libsystemd_tests += [
                 'sources' : files('sd-journal/test-journal-enum.c'),
                 'timeout' : 360,
         },
+        {
+                'sources' : files('sd-event/test-event.c'),
+                'timeout' : 120,
```

Comment:
Maybe, better to skip some tests or reduce iterations when slow test is disabled?

Reply:
it takes 6 seconds on my machine so it's not that slow, better to keep coverage up and increase the timeout for slow builders

Reply:
For me there is a difference - it should be OK to run all the tests, including in 'slow' mode, even on a bit older machine, and the default timeout value should have some reasonable margin. On my laptop with our default build system setup (inside a kvm VM) this test takes ~27s and very close to the 30s timeout of the 'simple tests'. And since all the other test classified as simple take usually less than 1 second I think it makes sense to increase the timeout for this one.

Reply:
OK.